src/components/FileUpload.test.tsx

```
@@ -0,0 +1,593 @@
+// Mock Admin Config Service
+const mockAdminConfigService = {
+  getLLMConfig: jest.fn(),
+  getPromptConfig: jest.fn(),
+};
+
+jest.mock('@/services/AdminConfigService', () => ({
+  AdminConfigService: jest.fn().mockImplementation(() => mockAdminConfigService),
+}));
+
+// Helper function to create mock files
+const createMockFile = (name: string, size: number, type: string = 'application/pdf') => {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)